src/System.Buffers/ref/System.Buffers.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Buffers
+{
+    public abstract class ArrayPool<T>
+    {
+        public const int DefaultNumberOfArraysPerBucket = 50;
+        public const int DefaultArrayLength = 1024000;
+
+        public static ArrayPool<T> Shared{ get; }
+        public static ArrayPool<T> Create(int maxArrayLength = DefaultArrayLength, int arraysPerBucket = DefaultNumberOfArraysPerBucket);
+        public abstract int RentedBufferCount { get; }
+        public abstract int TotalBufferCount { get; }
```

Comment:
RentedCount is for leak detection and TotalCount is for usage statistics, such as how utilized is the pool. It would be useful for tweaking pool values to figure out how accurate your configuration is...for example, am I utilizing 100% of the pool under high load? 50%? These values can help better fine tune the pool...or at least, that's the thought behind them :)